packages/contentful/migrations/pages/5v6w5j61tndm-add-pages-1670515365516.js

```diff
@@ -0,0 +1,119 @@
+module.exports.description = 'Adds content type for pages.';
+
+module.exports.up = (migration) => {
+  const pages = migration
+    .createContentType('pages')
+    .name('Pages')
+    .description('')
+    .displayField('title');
+  pages
+    .createField('title')
+    .name('Title')
+    .type('Text')
+    .localized(false)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  pages
+    .createField('path')
+    .name('Path')
+    .type('Symbol')
+    .localized(false)
+    .required(true)
+    .validations([
+      {
+        unique: true,
+      },
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  pages
+    .createField('shortText')
+    .name('Short Text')
+    .type('Text')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  pages
+    .createField('text')
+    .name('Text')
+    .type('RichText')
+    .localized(false)
+    .required(true)
```

Comment:
this is not marked as required in squidex, although it might be better to let it as required. Just pointing it out in case this was not the intention

Reply:
yeah i've actually made small amendments here